package.json

```diff
@@ -26,9 +26,6 @@
     "autoprefixer": "^10.4.19",
     "@mdn/browser-compat-data": "~5.5.28"
   },
-  "resolutions": {
-    "bun-types": "workspace:packages/bun-types"
-  },
```

Comment:
i think this is needed so @types/bun wont install them from npm and instead use local

![image](https://github.com/user-attachments/assets/1efc48cb-8a66-49ef-a3be-fcbe865748fe)